pkgs/top-level/python-packages.nix

```diff
@@ -173,6 +173,15 @@ in {
     };
   };
 
+  base58 = buildPythonPackage rec {
```

Comment:
Oh yes definitely, I would prefer it that way.  I thought things were moving in the opposite direction!